applications/dashboard/controllers/class.rolecontroller.php

```diff
@@ -228,6 +228,29 @@ public function index($roleID = null) {
             $this->setData('Roles', [$Role]);
         }
 
+        // Grab the total users for each role.
+        if (is_array($this->data('Roles'))) {
+            $pastThreshold = Gdn::userModel()->pastUserMegaThreshold();
+            $thresholdTypeExceptions = [
```

Comment:
I feel like it would be safer/cleaner to use RoleModel::getDefaultTypes() and unset RoleModel::TYPE_MEMBER. I don't like re-enumerating arbitrary lists like this unless it's required.